src/main/java/user_screens/CreateRegisterUserView.java

```diff
@@ -39,4 +40,8 @@ public CreateRegisterUserView(RegisterUserController registerusercontroller) {
         this.guestUser = new GuestUser();
     }
 
+    @Override
+    public void actionPerformed(ActionEvent e) {
+
+    }
 }
```

Comment:
I believe this (and the next few) are Henry's parts as well, so I'm not sure if this should be included in the commit unless it's something you both worked on.